### PR TITLE
fix: hardware label on wallet screen

### PR DESCRIPTION
**Pull request by fstream9**
**Description**

Adds hardware label to account info and overview components

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Progresses #???

### CODE REVIEW

File: app/util/address/index.js [hunk 1]
```diff
@@ -106,6 +106,24 @@ export function isQRHardwareAccount(address) {
 	return qrAccounts.includes(address.toLowerCase());
 }
 
+/**
+ * judge address is hardware account or not
+ *
+ * @param {String} address - String corresponding to an address
+ * @returns {Boolean} - Returns a boolean
+ */
+export function isHardwareAccount(address) {
```
Comment (zoltanarvai) [inline app/util/address/index.js:115]: Could `isQRHardwareAccount` be removed? or is it used at other locations too?

File: app/util/address/index.js [hunk 2]
```diff
@@ -106,6 +106,24 @@ export function isQRHardwareAccount(address) {
...
+	const { KeyringController } = Engine.context;
+	const { keyrings } = KeyringController.state;
+	for (const keyring of keyrings) {
+		if ([KeyringTypes.qr, KeyringTypes.ledger].includes(keyring.type) && keyring.accounts.includes(address)) {
```
Comment (zoltanarvai) [inline app/util/address/index.js:119]: casing of the address my differ... use lowercase comparison

**Review by zoltanarvai — CHANGES_REQUESTED** [review]

**Review by GummyGod — APPROVED** [review]

File: app/util/address/index.js (hunk 1 above)
Comment (fstream9) [reply app/util/address/index.js:115]: used elsewhere by QR UI

File: app/util/address/index.js (hunk 2 above)
Comment (fstream9) [reply app/util/address/index.js]: resolved 👍

**Review by zoltanarvai — APPROVED** [review]

**Pull request merged**